xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextSourceDirectorySet.xtend

```diff
@@ -27,4 +27,12 @@ interface XtextSourceDirectorySet extends PatternFilterable {
 	def void output(Closure<?> configureClosure)
 
 	def String getGeneratorTaskName()
+
+	def String getXtextToolingConfigurationName() {
+		getXtextToolingConfigurationName("xtextTooling")
+	}
+
+	def String getXtextToolingConfigurationName(String prefix) {
```

Comment:
How about we only keep this method and call it something like "qualifyConfigurationName"?

Reply:
will change later today